doc/reference/devices_gpu.md

```diff
@@ -51,8 +51,96 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
+
+Initialize a LXD container:
+
+    lxc init ubuntu:24.04 t1 --config security.nested=true --config security.privileged=true
+
+Add an iGPU device to your container:
+
+    lxc config device add t1 igpu0 gpu gputype=physical id=nvidia.com/gpu=igpu0
+
+Apply a `cloud-init` script to your instance to install the the Docker runtime, the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit) and a script to run a test [TensorRT](https://github.com/NVIDIA/TensorRT) workload:
+
+```yaml
+#cloud-config
+package_update: true
+packages:
+  - docker.io
+write_files:
+  - path: /etc/docker/daemon.json
+    permissions: '0644'
+    owner: root:root
+    content: |
+      {
+        "max-concurrent-downloads": 12,
+        "max-concurrent-uploads": 12,
+        "runtimes": {
+          "nvidia": {
+            "args": [],
+            "path": "nvidia-container-runtime"
+          }
+        }
+      }
+  - path: /root/run_tensorrt.sh
+    permissions: '0755'
+    owner: root:root
+    content: |
+      #!/bin/bash
+      echo "OS release,Kernel version"
+      (. /etc/os-release; echo "${PRETTY_NAME}"; uname -r) | paste -s -d,
+      echo
+      nvidia-smi -q
+      echo
+      exec bash -o pipefail -c "
+      cd /workspace/tensorrt/samples
+      make -j4
+      cd /workspace/tensorrt/bin
+      ./sample_onnx_mnist
+      retstatus=\${PIPESTATUS[0]}
+      echo \"Test exited with status code: \${retstatus}\" >&2
+      exit \${retstatus}
+      "
+runcmd:
+  - systemctl start docker
+  - systemctl enable docker
+  - usermod -aG docker root
+  - curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg
+  - curl -s -L https://nvidia.github.io/libnvidia-container/stable/deb/nvidia-container-toolkit.list | sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+  - apt-get update
+  - DEBIAN_FRONTEND=noninteractive apt-get install -y nvidia-container-toolkit
+  - nvidia-ctk runtime configure
+  - systemctl restart docker
+```
+
+Apply this `cloud-init` setup to your instance:
+
+    lxc config set t1 cloud-init.user-data - < cloud-init.yml
+
+Now you can start the instance:
+
+    lxc start t1
+
+Wait for the `cloud-init` process to finish:
+
+    lxc exec t1 -- cloud-init status --wait
+
+Finally, you can run your inference job inside the LXD container. Note: do not forget to modify the `mode` of the NVIDIA Container Runtime inside the LXD container to the value `csv` and not `auto` if you want to let Docker know that the NVIDIA runtime must be enabled with CSV mode. This configuration file can be found at `/etc/nvidia-container-runtime/config.toml`:
+
+    lxc shell t1
+    root@t1 # docker run --gpus all --runtime nvidia --rm -v $(pwd):/sh_input nvcr.io/nvidia/tensorrt:24.02-py3-igpu bash /sh_input/run_tensorrt.sh
```

Comment:
````suggestion
You can now run your inference job inside the LXD container. 

```{note}
The NVIDIA runtime must be enabled with CSV mode.
Therefore, you must modify the `mode` of the NVIDIA Container Runtime inside the LXD container to the value `csv` and not `auto` when running Docker.
You can do this by editing the `/etc/nvidia-container-runtime/config.toml` configuration file:

    lxc shell t1
    root@t1 # docker run --gpus all --runtime nvidia --rm -v $(pwd):/sh_input nvcr.io/nvidia/tensorrt:24.02-py3-igpu bash /sh_input/run_tensorrt.sh
```
````

Reply:
My suggestion is probably wrong, because I don't understand what you're trying to say here. Mainly I don't understand how the configuration file is related to the command you give, and what that has to do with CSV mode.

To explain it better, you should first explain what the problem is (in my guess, "The NVIDIA runtime must be enabled with CSV mode."), then give an explanation on how to fix it ("Therefore, you must modify the `mode` of the NVIDIA Container Runtime inside the LXD container to the value `csv` and not `auto` when running Docker.") and then give the concrete instructions ("You can do this by editing the `/etc/nvidia-container-runtime/config.toml` configuration file: <commands>").

Reply:
fixed